roles/common_tools/tasks/main.yml

```diff
@@ -39,16 +39,20 @@
     - python-pip
   when: releasever > 30
 
+- name: Install distgen package
```

Comment:
I think we do not really need distgen in TMT plans.
The distgen-generated files should be at this point correctly added to the repository.

Reply:
Fewer packages in TMT plans are always great. I will remote it.

Reply:
Resolved by https://github.com/sclorg/sclorg-testing-farm/pull/47/commits/1bbb41b40628b6885d2a7d38822539996d99b32d